projects/ion/src/lib/table/utilsTable.ts

```diff
@@ -74,17 +81,21 @@ export interface ActionConfirm<RowType> {
   cancelText?: string;
 }
 
+export type ActionPopover = PopoverProps;
+
 export interface ActionTable<RowType = SafeAny> {
   label: string;
   icon: string;
   disabled?: (row: RowType) => boolean;
   danger?: boolean;
   show?: (row: RowType) => boolean;
   call?: (row: RowType) => void;
+  secundCall?: (row: RowType) => void;
```

Comment:
remember to change other places too
```suggestion
  secondCall?: (row: RowType) => void;
```